input/pages/using-cql.md

```diff
@@ -159,9 +159,14 @@ Conformance Requirement 2.6 describes how to specify a code system within a CQL
 {: #conformance-requirement-2-6}
 
 1. Within CQL, the identifier of any code system reference **SHALL** be specified using a URI for the code system.
-2. The URI **SHALL** be the canonical URL for the code system
-3. The Code System declaration **MAY** include a version, consistent with the URI specification for FHIR and the code system
+<div class="new-content">
 
+2. The URI **SHALL** be the canonical URL for the code system.
+</div>
+<div class="new-content">
+
+3. The Code System declaration **MAY** include a version, consistent with the URI specification for FHIR and the code system.
+</div>
```

Comment:
I wouldn't call this new content? This was just fixing a period, can we remove the New Content designation here?